### PR TITLE
Add say_when to release episodes

### PR DESCRIPTION
**Pull request by kookster**
* Add say_when gem
* Init scheduled episode release in say_when init
* Move episode release method to the `Episode` resource

### CODE REVIEW

File: config/initializers/say_when.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+require 'say_when'
+require 'say_when/poller/concurrent_poller'
+
+# you can specify a the logger
```
Comment (gcampo88) [inline config/initializers/say_when.rb]: same typo (a+the) as in crier (from the Readme of the gem)

File: config/initializers/say_when.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  options[:storage_strategy]   = :active_record
+
+  # options[:processor_strategy] = :simple
+  options[:processor_strategy] = :active_job
+
+  options[:queue] = :feeder_default
+end
+
+begin
+  job = SayWhen.schedule(
```
Comment (gcampo88) [inline config/initializers/say_when.rb]: do you ever use `job` again? (and where is hound on this? :dog:)

File: config/initializers/say_when.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
+require 'say_when'
+# require 'say_when/poller/concurrent_poller'
+
+# you can specify a the logger
```
Comment (gcampo88) [inline config/initializers/say_when.rb]: same typo as in crier (a + the, from the gem readme)

File: config/initializers/say_when.rb (hunk 2 above)
Comment (kookster) [inline config/initializers/say_when.rb]: you don't - I was probably assigning it by habit, and b/c at one point I was logging it there

File: config/initializers/say_when.rb (hunk 3 above)
Comment (kookster) [inline config/initializers/say_when.rb]: thanks, fixing!

File: config/initializers/say_when.rb (hunk 2 above)
Comment (kookster) [inline config/initializers/say_when.rb]: (fixing this too.)

**Pull request merged**